mk-modules.nix

```diff
@@ -1,4 +1,5 @@
-flakeInputs: mkModulesOpts: let
+flakeInputs: mkModulesOpts:
```

Comment:
Which formatter did you use?

Reply:
The same one we use on the main code base, I'm fairly sure. I'll create a ticket so we add treefmt/a fmt check to this repo as well.